Gemfile

```diff
@@ -12,6 +12,8 @@ group :development do
   gem 'rubocop',       '~> 0.37.0'
   gem 'yard',          '~> 0.8.7'
   gem 'simplecov',     '~> 0.11.1'
+  gem 'mutant',        '~> 0.8.10'
```

Comment:
This should not be needed. It'll get pulled to the latest compatible version by `mutant-rspec` anyway.